### ISSUE TITLE
Error with todos example app and example tests

### ISSUE THREAD

**Issue by liangsun** [xolvio/meteor-cucumber#175]
When starting the todos example app with the example tests the following error can happen:

```
Error: Promise was fulfilled but got rejected with the following reason: Error: stale element reference: element is not attached to the page document
  at new ErrorHandler.CommandError (/Users/liangsun/.meteor/packages/xolvio_cucumber/.0.12.3.kt41ck++os+web.browser+web.cordova/npm/node_modules/chimp/node_modules/webdriverio/lib/utils/ErrorHandler.js:86:12)
  at Object.<anonymous> (/Users/liangsun/.meteor/packages/xolvio_cucumber/.0.12.3.kt41ck++os+web.browser+web.cordova/npm/node_modules/chimp/node_modules/webdriverio/lib/commands/waitUntil.js:94:22)
  at Object.safeExecute (/Users/liangsun/.meteor/packages/xolvio_cucumber/.0.12.3.kt41ck++os+web.browser+web.cordova/npm/node_modules/chimp/node_modules/webdriverio/lib/utils/safeExecute.js:22:24)
  at Object.WebdriverIO.resolve (/Users/liangsun/.meteor/packages/xolvio_cucumber/.0.12.3.kt41ck++os+web.browser+web.cordova/npm/node_modules/chimp/node_modules/webdriverio/lib/webdriverio.js:73:29)
  at /Users/liangsun/.meteor/packages/xolvio_cucumber/.0.12.3.kt41ck++os+web.browser+web.cordova/npm/node_modules/chimp/node_modules/webdriverio/lib/webdriverio.js:193:32
  at _rejected (/Users/liangsun/.meteor/packages/xolvio_cucumber/.0.12.3.kt41ck++os+web.browser+web.cordova/npm/node_modules/chimp/node_modules/webdriverio/node_modules/q/q.js:830:24)
  at /Users/liangsun/.meteor/packages/xolvio_cucumber/.0.12.3.kt41ck++os+web.browser+web.cordova/npm/node_modules/chimp/node_modules/webdriverio/node_modules/q/q.js:856:30
  at Promise.when (/Users/liangsun/.meteor/packages/xolvio_cucumber/.0.12.3.kt41ck++os+web.browser+web.cordova/npm/node_modules/chimp/node_modules/webdriverio/node_modules/q/q.js:1108:31)
  at Promise.promise.promiseDispatch (/Users/liangsun/.meteor/packages/xolvio_cucumber/.0.12.3.kt41ck++os+web.browser+web.cordova/npm/node_modules/chimp/node_modules/webdriverio/node_modules/q/q.js:774:41)
  at /Users/liangsun/.meteor/packages/xolvio_cucumber/.0.12.3.kt41ck++os+web.browser+web.cordova/npm/node_modules/chimp/node_modules/webdriverio/node_modules/q/q.js:590:44
  at runSingle (/Users/liangsun/.meteor/packages/xolvio_cucumber/.0.12.3.kt41ck++os+web.browser+web.cordova/npm/node_modules/chimp/node_modules/webdriverio/node_modules/q/q.js:123:13)
  at flush (/Users/liangsun/.meteor/packages/xolvio_cucumber/.0.12.3.kt41ck++os+web.browser+web.cordova/npm/node_modules/chimp/node_modules/webdriverio/node_modules/q/q.js:111:13)
  at process._tickDomainCallback (node.js:486:13)
```

Herer is how to reproduce the bug

```
meteor create --example todos
cd todos
meteor add xolvio:cucumber
VELOCITY_DEBUG=1 meteor
```
Then click the `Add Sample Test` at the web and restart the meteor app.

The above error will show.

The console output:

```
liangsun@1iangMac$  VELOCITY_DEBUG=1 meteor
[[[[[ ~/todos ]]]]]                           

=> Started proxy.                             
=> Started MongoDB.                           
I20150728-08:56:52.360(8)? [velocity] adding velocity core
I20150728-08:56:52.362(8)? [xolvio:cucumber] Cucumber hub is loading
I20150728-08:56:52.362(8)? [velocity] Register framework cucumber with regex ^tests/cucumber/((?!node_modules/).)+\.(feature|js|coffee|litcoffee|coffee\.md)$
I20150728-08:56:53.370(8)? [velocity] Server startup
I20150728-08:56:53.371(8)? [velocity] app dir /Users/liangsun/todos
I20150728-08:56:53.371(8)? [velocity] config = {
I20150728-08:56:53.371(8)?   "cucumber": {
I20150728-08:56:53.371(8)?     "regex": "^tests/cucumber/((?!node_modules/).)+\\.(feature|js|coffee|litcoffee|coffee\\.md)$",
I20150728-08:56:53.371(8)?     "name": "cucumber",
I20150728-08:56:53.371(8)?     "_regexp": {}
I20150728-08:56:53.371(8)?   }
I20150728-08:56:53.371(8)? }
I20150728-08:56:53.372(8)? [velocity] resetting the world
I20150728-08:56:53.372(8)? [velocity] frameworks with disable auto reset: []
I20150728-08:56:53.377(8)? [velocity] Add paths to watcher [ '/Users/liangsun/todos/tests' ]
I20150728-08:56:53.380(8)? [velocity] 1 mirror(s) requested
I20150728-08:56:53.398(8)? [velocity] Mirror process forked with pid 79197
I20150728-08:56:53.399(8)? [velocity] cucumber is starting a mirror at http://localhost:58791/.
I20150728-08:56:53.402(8)? [velocity] File scan complete, now watching /tests
I20150728-08:56:53.402(8)? [velocity] Triggering queued startup functions
I20150728-08:56:53.418(8)? [velocity] You can see the mirror logs at: tail -f /Users/liangsun/todos/.meteor/local/log/cucumber.log
=> Started your app.

=> App running at: http://localhost:3000/
I20150728-08:57:02.138(8)? [velocity] found  4 sample test files for cucumber
I20150728-08:57:02.152(8)? [velocity] resetting cucumber
I20150728-08:57:02.281(8)? [velocity] Search framework for path tests/cucumber/package.json
I20150728-08:57:02.286(8)? [velocity] No framework registered for tests/cucumber/package.json
I20150728-08:57:02.386(8)? [velocity] Search framework for path tests/cucumber/features/sample.feature
I20150728-08:57:02.386(8)? [velocity] Target framework for tests/cucumber/features/sample.feature is cucumber
I20150728-08:57:02.392(8)? [velocity] Search framework for path tests/cucumber/fixtures/my_fixture.js
I20150728-08:57:02.394(8)? [velocity] Target framework for tests/cucumber/fixtures/my_fixture.js is cucumber
I20150728-08:57:02.554(8)? [velocity] Search framework for path tests/cucumber/features/step_definitions/sample_steps.js
I20150728-08:57:02.554(8)? [velocity] Target framework for tests/cucumber/features/step_definitions/sample_steps.js is cucumber
I20150728-08:57:16.699(8)? [velocity] Mirror registered. Handshaking with mirror...
I20150728-08:57:16.725(8)? [velocity] Connected to mirror, setting state to ready { framework: 'cucumber',
I20150728-08:57:16.726(8)?   host: 'http://localhost:58791/',
I20150728-08:57:16.726(8)?   port: '58791' }
I20150728-08:57:25.868(8)? [velocity] Mirror registered. Handshaking with mirror...
I20150728-08:57:25.883(8)? [velocity] Connected to mirror, setting state to ready { framework: 'cucumber',
I20150728-08:57:25.883(8)?   host: 'http://localhost:58791/',
I20150728-08:57:25.883(8)?   port: '58791' }
```

content of `.meteor/local/log/cucumber.log`

```
Spawn script arguments:
parentPid: 79174
taskName: cucumber
command: meteor
commandArguments: [ 'run',
  '--test-app',
  '--port',
  '58791',
  '--include-tests',
  'cucumber/fixtures',
  '--settings',
  '/var/folders/fn/y1jvjgdn7v7g2y0ntvdtjwxc0000gn/T/tmp-79195V8yPso44neVc.tmp',
  '--raw-logs',
  '--release',
  'velocity:METEOR@1.1.0.2_3' ]
pid: 79197
child.pid: 79198
=> Using velocity:METEOR@1.1.0.2_3 as requested (overriding Meteor 1.1.0.2)

[[[[[ ~/todos ]]]]]

=> Started proxy.
[sanjo:long-running-child-process]: Exposing imports for testing purposes
[velocity] adding velocity core
[xolvio:cucumber] Mirror server is initializing
[xolvio:cucumber] Starting Chimp
[xolvio:cucumber] Screenshots dir is /Users/liangsun/todos/tests/cucumber/.screenshots
[xolvio:cucumber] Starting Chimp with /Users/liangsun/.meteor/packages/xolvio_cucumber/.0.12.3.kt41ck++os+web.browser+web.cordova/npm/node_modules/chimp/bin/chimp [ '/Users/liangsun/.meteor/packages/xolvio_cucumber/.0.12.3.kt41ck++os+web.browser+web.cordova/npm/node_modules/chimp/bin/chimp',
  '-r',
  '/Users/liangsun/todos/tests/cucumber/features',
  '--snippets',
  '--ddp=http://localhost:58791/',
  '--log=error',
  '--debug',
  '--screenshotsPath=/Users/liangsun/todos/tests/cucumber/.screenshots',
  '--tags=@dev',
  '--server',
  '--serverPort=8866' ] { cwd: '/Users/liangsun/todos/tests/cucumber',
  stdio: 'inherit',
  env: 
   { METEOR_SHELL_DIR: '/private/var/folders/fn/y1jvjgdn7v7g2y0ntvdtjwxc0000gn/T/meteor-test-run1qrl6zu/.meteor/local/shell',
     TERM_PROGRAM: 'Apple_Terminal',
     rvm_bin_path: '/Users/liangsun/.rvm/bin',
     MIRROR_PORT: '58791',
     SHELL: '/bin/bash',
     TERM: 'xterm-256color',
     IS_MIRROR: 'true',
     HOST: 'http://localhost:58791/',
     CLICOLOR: '1',
     TMPDIR: '/var/folders/fn/y1jvjgdn7v7g2y0ntvdtjwxc0000gn/T/',
     Apple_PubSub_Socket_Render: '/private/tmp/com.apple.launchd.viccC0IZcl/Render',
     TERM_PROGRAM_VERSION: '343.7',
     HANDSHAKE: 'true',
     OLDPWD: '/Users/liangsun/.meteor/packages/velocity_meteor-tool/1.1.3_4/mt-os.osx.x86_64',
     MOBILE_DDP_URL: 'http://localhost:58791',
     TERM_SESSION_ID: '8F31EC75-E884-4918-9E7D-381B5602D1EA',
     GIT_EDITOR: 'vim',
     USER: 'liangsun',
     _system_type: 'Darwin',
     RBENV_ROOT: '/usr/local/var/rbenv',
     rvm_path: '/Users/liangsun/.rvm',
     TERM_SESSION_CLASS_ID: '360A3622-5A50-4943-AACC-9B564A5F6158',
     SSH_AUTH_SOCK: '/private/tmp/com.apple.launchd.4LyULiBi4K/Listeners',
     __CF_USER_TEXT_ENCODING: '0x1F5:0x0:0x0',
     rvm_prefix: '/Users/liangsun',
     PATH: '/Users/liangsun/.meteor/packages/velocity_meteor-tool/.1.1.3_4.1981n91++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/bin:/Users/liangsun/.meteor/packages/meteor-tool/.1.1.3.1wysac9++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/bin:/usr/local/var/rbenv/shims:/usr/local/heroku/bin:/usr/local/var/rbenv/shims:/usr/local/heroku/bin:/usr/local/var/rbenv/shims:/usr/local/heroku/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/opt/X11/bin:/usr/local/git/bin:/usr/texbin:/Users/liangsun/.rvm/bin:/Users/liangsun/.rvm/bin:/Users/liangsun/.rvm/bin:/Users/liangsun/.rvm/bin',
     METEOR_SETTINGS: '{"title":"test todos"}',
     PWD: '/Users/liangsun/todos',
     PARENT_URL: 'http://localhost:3000/',
     MONGO_URL: 'mongodb://127.0.0.1:3001/cucumber',
     EDITOR: 'vim',
     LANG: 'en_US.UTF-8',
     PYTHONSTARTUP: '/Users/liangsun/.pythonrc.py',
     NODE_PATH: '',
     XPC_FLAGS: '0x0',
     _system_arch: 'x86_64',
     NODE_ENV: 'development',
     _system_version: '10.10',
     ROOT_URL: 'http://localhost:58791/',
     APP_ID: 'cboji614b18u11hgvhfz',
     XPC_SERVICE_NAME: '0',
     rvm_version: '1.26.11 (latest)',
     METEOR_PARENT_PID: '79198',
     SHLVL: '1',
     HOME: '/Users/liangsun',
     VELOCITY_DEBUG: '1',
     METEOR_PRINT_ON_LISTEN: 'true',
     LOGNAME: 'liangsun',
     FRAMEWORK: 'cucumber',
     HTTP_FORWARDED_COUNT: '2',
     PORT: '24301',
     MONGO_OPLOG_URL: 'mongodb://127.0.0.1:3001/local',
     VELOCITY_MAIN_APP_PATH: '/Users/liangsun/todos',
     ROOT_URL_PATH: '',
     SECURITYSESSIONID: '186a5',
     _system_name: 'OSX',
     MOBILE_ROOT_URL: 'http://localhost:58791' } }
[xolvio:cucumber] Chimp process forked with pid 79228
=> Started your app.

=> App running at: http://localhost:58791/
[velocity] Mirror started. Connecting via DDP to parent
[velocity] Mirror connected to parent. Registering mirror...
Spawn script arguments:
parentPid: 79198
taskName: Chimp_8866
command: /Users/liangsun/.meteor/packages/velocity_meteor-tool/.1.1.3_4.1981n91++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/bin/node
commandArguments: [ '/Users/liangsun/.meteor/packages/xolvio_cucumber/.0.12.3.kt41ck++os+web.browser+web.cordova/npm/node_modules/chimp/bin/chimp',
  '-r',
  '/Users/liangsun/todos/tests/cucumber/features',
  '--snippets',
  '--ddp=http://localhost:58791/',
  '--log=error',
  '--debug',
  '--screenshotsPath=/Users/liangsun/todos/tests/cucumber/.screenshots',
  '--tags=@dev',
  '--server',
  '--serverPort=8866' ]
pid: 79228
child.pid: 79229
[velocity] Established connection with Velocity.
[chimp] Running /Users/liangsun/.meteor/packages/velocity_meteor-tool/.1.1.3_4.1981n91++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/bin/npm install
[chimp] stdout: underscore@1.8.3 node_modules/underscore

[chimp] stderr: 
[chimp] Chimp server is running on port 8866 http://localhost:58791/
[sanjo:long-running-child-process]: Exposing imports for testing purposes
[velocity] adding velocity core
[xolvio:cucumber] Mirror server is initializing
[xolvio:cucumber] Starting Chimp
[xolvio:cucumber] Chimp is already running in server mode. Initting
[xolvio:cucumber] Connecting to hub
[xolvio:cucumber] Connected to hub.
=> Meteor server restarted
[velocity] Mirror started. Connecting via DDP to parent
[velocity] Mirror connected to parent. Registering mirror...
[chimp] Handshaking with DDP server
[xolvio:cucumber] Received handshake from Chimp.
[xolvio:cucumber] Connecting to hub
[chimp] Handshake complete, closing DDP connection
[velocity] Established connection with Velocity.
[xolvio:cucumber] Connected to hub.
[xolvio:cucumber] Find and run triggered { '0': 
   { name: 'sample_steps.js',
     absolutePath: '/Users/liangsun/todos/tests/cucumber/features/step_definitions/sample_steps.js',
     relativePath: 'tests/cucumber/features/step_definitions/sample_steps.js',
     targetFramework: 'cucumber',
     lastModified: 1438045022554,
     _id: '/Users/liangsun/todos/tests/cucumber/features/step_definitions/sample_steps.js' } }
[xolvio:cucumber] Running in batch-features mode.
[xolvio:cucumber] Cucumber is running
[chimp] interrupting
[chimp] no processes to interrupt
[xolvio:cucumber] Find and run triggered { '0': 
   { name: 'sample_steps.js',
     absolutePath: '/Users/liangsun/todos/tests/cucumber/features/step_definitions/sample_steps.js',
     relativePath: 'tests/cucumber/features/step_definitions/sample_steps.js',
     targetFramework: 'cucumber',
     lastModified: 1438045022554,
     _id: '/Users/liangsun/todos/tests/cucumber/features/step_definitions/sample_steps.js' } }
[xolvio:cucumber] Running in batch-features mode.
[xolvio:cucumber] Cucumber is running
[chimp] rerunning
[chimp] Running...
[chimp] interrupting
[chimp] no processes to interrupt
[chimp][selenium] Selenium object created
[chimp][selenium] Start
[chimp][selenium] Installing Selenium + drivers if needed
[chimp] interrupting
[chimp] 2 processes to interrupt
[chimp][cucumber] interrupting cucumber
[chimp][cucumber] no child to interrupt
[chimp][selenium] interrupt called
[chimp][selenium] interrupt is not killing selenium because clean-selenium-server not set
[chimp] Finished interrupting processes
[chimp] rerunning
[chimp] Running...
[chimp] interrupting
[chimp] no processes to interrupt
[chimp][selenium] Selenium object already exists, not creating a new one
[chimp][selenium] Start
[chimp][selenium] Installing Selenium + drivers if needed

[chimp][selenium] hub can be seen at http://localhost:59021/wd/hub

[chimp][selenium] hub can be seen at http://localhost:59021/wd/hub
[chimp] Finished running async processes with errors
[chimp] finished rerun
[xolvio:cucumber] Bad response from Chimp server. [TypeError: Cannot read property 'message' of null]
[chimp] interrupting
[chimp] 2 processes to interrupt
[chimp][cucumber] interrupting cucumber
[chimp][cucumber] no child to interrupt
[chimp][selenium] interrupt called
[chimp][selenium] interrupt is not killing selenium because clean-selenium-server not set
[chimp] Finished interrupting processes
[chimp][session-manager] options are { port: '59021', browser: 'chrome', deviceName: undefined }
[chimp][session-manager] created a new SessionManager { port: '59021', browser: 'chrome', deviceName: undefined }
[chimp][cucumber] Running with [ 'node',
  '/Users/liangsun/.meteor/packages/xolvio_cucumber/.0.12.3.kt41ck++os+web.browser+web.cordova/npm/node_modules/chimp/node_modules/.bin/cucumber.js',
  '-r',
  'features',
  '-r',
  '/Users/liangsun/.meteor/packages/xolvio_cucumber/.0.12.3.kt41ck++os+web.browser+web.cordova/npm/node_modules/chimp/lib/cucumberjs/world.js',
  '-r',
  '/Users/liangsun/.meteor/packages/xolvio_cucumber/.0.12.3.kt41ck++os+web.browser+web.cordova/npm/node_modules/chimp/lib/cucumberjs/hooks.js',
  '-r',
  '/Users/liangsun/todos/tests/cucumber/features',
  '--snippets',
  'true',
  '--tags',
  '@dev',
  '--format',
  'pretty' ]
[chimp][world] creating DDP connection

[chimp][hooks] Starting BeforeFeatures

[chimp][world] getting browser

[chimp][session-manager] options are { port: '59021', browser: 'chrome', deviceName: undefined }

[chimp][session-manager] created a new SessionManager { port: '59021', browser: 'chrome', deviceName: undefined }

[chimp][world] custom chimp.js not found, loading defaults

[chimp][world] webdriverOptions are  {"waitforTimeout":10000,"desiredCapabilities":{"browserName":"chrome","platform":"ANY","name":""},"host":"localhost","port":"59021","logLevel":"silent"}

[chimp][session-manager] creating webdriver remote 

[chimp][session-manager] checking connection to selenium server

[chimp][session-manager] requesting sessions from http://localhost:59021/wd/hub/sessions

[chimp][session-manager] received data {"sessionId":null,"status":0,"state":"success","value":[],"class":"org.openqa.selenium.remote.Response","hCode":1406035614}

[chimp][session-manager] Did not find any open selenium sessions, not reusing a session

[chimp][session-manager] monkey patching the browser object

[chimp][hooks] init browser

[chimp][session-manager] initializing browser

[chimp][hooks] setup DDP

[chimp][hooks] connecting via DDP to http://localhost:58791/

[chimp][hooks] init browser callback

[chimp][hooks] set timeoutsAsyncScript

[chimp][hooks] Finished BeforeFeatures
[chimp][hooks] User-defined BeforeFeatures not found, finishing up


Feature: One-liner description of this feature


  As a [role]
  I want [feature]
  So that [benefit]

  The story above is to set context for the reader. It doesn't actually have any impact on the test
  itself. The phrases inside the scenarios are ties to test code using regex, which you can see in
  /tests/features/step_definitions/steps.js

  In this textual part of the file, you can include context about this feature, you can add links to
  external documents and whatever is needed to create the full specification.


[chimp][world] setting up world

  @dev
  Scenario: This scenario will run on both dev and CI   # features/sample.feature:20

    Given I am a new user                               # features/sample.feature:16

    When I navigate to "/"                              # features/sample.feature:21

    Then I should see the title "intentional failure"   # features/sample.feature:22
      Error: Promise was fulfilled but got rejected with the following reason: Error: stale element reference: element is not attached to the page document
        at new ErrorHandler.CommandError (/Users/liangsun/.meteor/packages/xolvio_cucumber/.0.12.3.kt41ck++os+web.browser+web.cordova/npm/node_modules/chimp/node_modules/webdriverio/lib/utils/ErrorHandler.js:86:12)
        at Object.<anonymous> (/Users/liangsun/.meteor/packages/xolvio_cucumber/.0.12.3.kt41ck++os+web.browser+web.cordova/npm/node_modules/chimp/node_modules/webdriverio/lib/commands/waitUntil.js:94:22)
        at Object.safeExecute (/Users/liangsun/.meteor/packages/xolvio_cucumber/.0.12.3.kt41ck++os+web.browser+web.cordova/npm/node_modules/chimp/node_modules/webdriverio/lib/utils/safeExecute.js:22:24)
        at Object.WebdriverIO.resolve (/Users/liangsun/.meteor/packages/xolvio_cucumber/.0.12.3.kt41ck++os+web.browser+web.cordova/npm/node_modules/chimp/node_modules/webdriverio/lib/webdriverio.js:73:29)
        at /Users/liangsun/.meteor/packages/xolvio_cucumber/.0.12.3.kt41ck++os+web.browser+web.cordova/npm/node_modules/chimp/node_modules/webdriverio/lib/webdriverio.js:193:32
        at _rejected (/Users/liangsun/.meteor/packages/xolvio_cucumber/.0.12.3.kt41ck++os+web.browser+web.cordova/npm/node_modules/chimp/node_modules/webdriverio/node_modules/q/q.js:830:24)
        at /Users/liangsun/.meteor/packages/xolvio_cucumber/.0.12.3.kt41ck++os+web.browser+web.cordova/npm/node_modules/chimp/node_modules/webdriverio/node_modules/q/q.js:856:30
        at Promise.when (/Users/liangsun/.meteor/packages/xolvio_cucumber/.0.12.3.kt41ck++os+web.browser+web.cordova/npm/node_modules/chimp/node_modules/webdriverio/node_modules/q/q.js:1108:31)
        at Promise.promise.promiseDispatch (/Users/liangsun/.meteor/packages/xolvio_cucumber/.0.12.3.kt41ck++os+web.browser+web.cordova/npm/node_modules/chimp/node_modules/webdriverio/node_modules/q/q.js:774:41)
        at /Users/liangsun/.meteor/packages/xolvio_cucumber/.0.12.3.kt41ck++os+web.browser+web.cordova/npm/node_modules/chimp/node_modules/webdriverio/node_modules/q/q.js:590:44
        at runSingle (/Users/liangsun/.meteor/packages/xolvio_cucumber/.0.12.3.kt41ck++os+web.browser+web.cordova/npm/node_modules/chimp/node_modules/webdriverio/node_modules/q/q.js:123:13)
        at flush (/Users/liangsun/.meteor/packages/xolvio_cucumber/.0.12.3.kt41ck++os+web.browser+web.cordova/npm/node_modules/chimp/node_modules/webdriverio/node_modules/q/q.js:111:13)
        at process._tickDomainCallback (node.js:486:13)




(::) failed steps (::)

Error: Promise was fulfilled but got rejected with the following reason: Error: stale element reference: element is not attached to the page document
  at new ErrorHandler.CommandError (/Users/liangsun/.meteor/packages/xolvio_cucumber/.0.12.3.kt41ck++os+web.browser+web.cordova/npm/node_modules/chimp/node_modules/webdriverio/lib/utils/ErrorHandler.js:86:12)
  at Object.<anonymous> (/Users/liangsun/.meteor/packages/xolvio_cucumber/.0.12.3.kt41ck++os+web.browser+web.cordova/npm/node_modules/chimp/node_modules/webdriverio/lib/commands/waitUntil.js:94:22)
  at Object.safeExecute (/Users/liangsun/.meteor/packages/xolvio_cucumber/.0.12.3.kt41ck++os+web.browser+web.cordova/npm/node_modules/chimp/node_modules/webdriverio/lib/utils/safeExecute.js:22:24)
  at Object.WebdriverIO.resolve (/Users/liangsun/.meteor/packages/xolvio_cucumber/.0.12.3.kt41ck++os+web.browser+web.cordova/npm/node_modules/chimp/node_modules/webdriverio/lib/webdriverio.js:73:29)
  at /Users/liangsun/.meteor/packages/xolvio_cucumber/.0.12.3.kt41ck++os+web.browser+web.cordova/npm/node_modules/chimp/node_modules/webdriverio/lib/webdriverio.js:193:32
  at _rejected (/Users/liangsun/.meteor/packages/xolvio_cucumber/.0.12.3.kt41ck++os+web.browser+web.cordova/npm/node_modules/chimp/node_modules/webdriverio/node_modules/q/q.js:830:24)
  at /Users/liangsun/.meteor/packages/xolvio_cucumber/.0.12.3.kt41ck++os+web.browser+web.cordova/npm/node_modules/chimp/node_modules/webdriverio/node_modules/q/q.js:856:30
  at Promise.when (/Users/liangsun/.meteor/packages/xolvio_cucumber/.0.12.3.kt41ck++os+web.browser+web.cordova/npm/node_modules/chimp/node_modules/webdriverio/node_modules/q/q.js:1108:31)
  at Promise.promise.promiseDispatch (/Users/liangsun/.meteor/packages/xolvio_cucumber/.0.12.3.kt41ck++os+web.browser+web.cordova/npm/node_modules/chimp/node_modules/webdriverio/node_modules/q/q.js:774:41)
  at /Users/liangsun/.meteor/packages/xolvio_cucumber/.0.12.3.kt41ck++os+web.browser+web.cordova/npm/node_modules/chimp/node_modules/webdriverio/node_modules/q/q.js:590:44
  at runSingle (/Users/liangsun/.meteor/packages/xolvio_cucumber/.0.12.3.kt41ck++os+web.browser+web.cordova/npm/node_modules/chimp/node_modules/webdriverio/node_modules/q/q.js:123:13)
  at flush (/Users/liangsun/.meteor/packages/xolvio_cucumber/.0.12.3.kt41ck++os+web.browser+web.cordova/npm/node_modules/chimp/node_modules/webdriverio/node_modules/q/q.js:111:13)
  at process._tickDomainCallback (node.js:486:13)


Failing scenarios:
/Users/liangsun/todos/tests/cucumber/features/sample.feature:20 # Scenario: This scenario will run on both dev and CI

1 scenario (1 failed)
3 steps (1 failed, 2 passed)
[chimp][hooks] Starting AfterFeatures
[chimp][hooks] Ending browser session
[chimp][session-manager] watch / server mode are true, not killing session
[chimp][hooks] Ended browser sessions
[chimp][hooks] Finished AfterFeatures

[chimp][cucumber] Closed with code 2
[chimp] finished rerun
[chimp] Finished running async processes with errors
Error: Promise was fulfilled but got rejected with the following reason: Error: stale element reference: element is not attached to the page document
  at new ErrorHandler.CommandError (/Users/liangsun/.meteor/packages/xolvio_cucumber/.0.12.3.kt41ck++os+web.browser+web.cordova/npm/node_modules/chimp/node_modules/webdriverio/lib/utils/ErrorHandler.js:86:12)
  at Object.<anonymous> (/Users/liangsun/.meteor/packages/xolvio_cucumber/.0.12.3.kt41ck++os+web.browser+web.cordova/npm/node_modules/chimp/node_modules/webdriverio/lib/commands/waitUntil.js:94:22)
  at Object.safeExecute (/Users/liangsun/.meteor/packages/xolvio_cucumber/.0.12.3.kt41ck++os+web.browser+web.cordova/npm/node_modules/chimp/node_modules/webdriverio/lib/utils/safeExecute.js:22:24)
  at Object.WebdriverIO.resolve (/Users/liangsun/.meteor/packages/xolvio_cucumber/.0.12.3.kt41ck++os+web.browser+web.cordova/npm/node_modules/chimp/node_modules/webdriverio/lib/webdriverio.js:73:29)
  at /Users/liangsun/.meteor/packages/xolvio_cucumber/.0.12.3.kt41ck++os+web.browser+web.cordova/npm/node_modules/chimp/node_modules/webdriverio/lib/webdriverio.js:193:32
  at _rejected (/Users/liangsun/.meteor/packages/xolvio_cucumber/.0.12.3.kt41ck++os+web.browser+web.cordova/npm/node_modules/chimp/node_modules/webdriverio/node_modules/q/q.js:830:24)
  at /Users/liangsun/.meteor/packages/xolvio_cucumber/.0.12.3.kt41ck++os+web.browser+web.cordova/npm/node_modules/chimp/node_modules/webdriverio/node_modules/q/q.js:856:30
  at Promise.when (/Users/liangsun/.meteor/packages/xolvio_cucumber/.0.12.3.kt41ck++os+web.browser+web.cordova/npm/node_modules/chimp/node_modules/webdriverio/node_modules/q/q.js:1108:31)
  at Promise.promise.promiseDispatch (/Users/liangsun/.meteor/packages/xolvio_cucumber/.0.12.3.kt41ck++os+web.browser+web.cordova/npm/node_modules/chimp/node_modules/webdriverio/node_modules/q/q.js:774:41)
  at /Users/liangsun/.meteor/packages/xolvio_cucumber/.0.12.3.kt41ck++os+web.browser+web.cordova/npm/node_modules/chimp/node_modules/webdriverio/node_modules/q/q.js:590:44
  at runSingle (/Users/liangsun/.meteor/packages/xolvio_cucumber/.0.12.3.kt41ck++os+web.browser+web.cordova/npm/node_modules/chimp/node_modules/webdriverio/node_modules/q/q.js:123:13)
  at flush (/Users/liangsun/.meteor/packages/xolvio_cucumber/.0.12.3.kt41ck++os+web.browser+web.cordova/npm/node_modules/chimp/node_modules/webdriverio/node_modules/q/q.js:111:13)
  at process._tickDomainCallback (node.js:486:13)

stream error Network error: ws://localhost:3000/websocket: connect ECONNREFUSED
stream error Network error: ws://localhost:3000/websocket: connect ECONNREFUSED
stream error Network error: ws://localhost:3000/websocket: connect ECONNREFUSED
stream error Network error: ws://localhost:3000/websocket: connect ECONNREFUSED
Parent process ( 79174 ) is dead! Exiting cucumber
```

content of `.meteor/versions`

```
accounts-base@1.2.0
accounts-password@1.1.1
amplify@1.0.0
autoupdate@1.2.1
base64@1.0.3
binary-heap@1.0.3
blaze@2.1.2
blaze-tools@1.0.3
boilerplate-generator@1.0.3
callback-hook@1.0.3
check@1.0.5
coffeescript@1.0.6
ddp@1.1.0
deps@1.0.7
ejson@1.0.6
email@1.0.6
fastclick@1.0.3
geojson-utils@1.0.3
html-tools@1.0.4
htmljs@1.0.4
http@1.1.0
id-map@1.0.3
insecure@1.0.3
iron:core@0.3.4
iron:dynamic-template@0.4.1
iron:layout@0.4.1
iron:router@0.9.4
jquery@1.11.3_2
json@1.0.3
launch-screen@1.0.2
less@1.0.14
livedata@1.0.13
localstorage@1.0.3
logging@1.0.7
meteor@1.1.6
meteor-platform@1.2.2
minifiers@1.1.5
minimongo@1.0.8
mobile-status-bar@1.0.3
mongo@1.1.0
npm-bcrypt@0.7.8_2
observe-sequence@1.0.6
ordered-dict@1.0.3
package-version-parser@3.0.3
practicalmeteor:chai@1.9.2_3
practicalmeteor:loglevel@1.1.0_3
random@1.0.3
reactive-dict@1.1.0
reactive-var@1.0.5
reload@1.1.3
retry@1.0.3
routepolicy@1.0.5
sanjo:long-running-child-process@1.1.1
sanjo:meteor-files-helpers@1.1.0_7
sanjo:meteor-version@1.0.0
service-configuration@1.0.4
session@1.1.0
sha@1.0.3
spacebars@1.0.6
spacebars-compiler@1.0.6
srp@1.0.3
templating@1.1.1
tracker@1.0.7
ui@1.0.6
underscore@1.0.3
url@1.0.4
velocity:chokidar@1.0.3_1
velocity:core@0.7.1
velocity:html-reporter@0.7.3
velocity:meteor-internals@1.1.0_7
velocity:shim@0.1.0
webapp@1.2.0
webapp-hashing@1.0.3
xolvio:cucumber@0.12.3
```

content of `.meteor/packages`

```
# Meteor packages used by this project, one per line.
# Check this file (and the other files in this directory) into your repository.
#
# 'meteor add' and 'meteor remove' will edit this file for you,
# but you can also edit it by hand.

meteor-platform
insecure
less
iron:router@0.9.4
accounts-password
xolvio:cucumber
```

**Comment by Sanjo** [comment]
Hi, I can reproduce the error without the settings. I will edit you post. Thanks for the nice structured report btw. :-)

**Comment by Sanjo** [comment]
The error boils down to the command: `this.client.waitForVisible('body *')`. I suspect it is the wildcard selector.

**Comment by liangsun** [comment]
:+1: :beers: Thanks for this quick fix.

**Comment by timurbazhirov** [comment]
@Sanjo - thanks for all the great work on velocity. I've just faced the same error as Liang did above and noted that it looks like the sample tests steps that come with this repo still have `waitForVisible('body *')` instead of `waitForExist('title')` both in [0.14-rc](https://github.com/xolvio/meteor-cucumber/blob/0.14.0-rc.x/src/sample-tests/steps.js) and in [master](https://github.com/xolvio/meteor-cucumber/blob/0.14.0-rc.x/src/sample-tests/steps.js) branches.

**Comment by Sanjo** [comment]
@timurbazhirov Thanks for pointing that out. I have fixed it now in master and 0.14-rc
